docker/docker-compose.yml

```diff
@@ -19,8 +19,11 @@ services:
       # Do not edit the next line. If you want to change the media storage location on your system, edit the value of UPLOAD_LOCATION in the .env file
       - ${UPLOAD_LOCATION}:/usr/src/app/upload
       - /etc/localtime:/etc/localtime:ro
-    env_file:
-      - .env
+    environment:
+      DB_USERNAME: ${DB_USERNAME}
+      DB_PASSWORD: ${DB_PASSWORD}
+      DB_USERNAME: ${DB_DATABASE_NAME}
+      TZ: $TZ
```

Comment:
This probably needs to be ${TZ}, no?

Reply:
Yeah, it technically doesn't *need* to be but for consistency sake it should be. I had made that change locally but forgot to commit.